Ports/README.md

```diff
@@ -276,7 +277,7 @@ is placed in.
 
 #### `prefix`
 
-The location of the ports directory, only used for the `package.db` file for
+The location of the portsʼ directory, only used for the `package.db` file for
```

Comment:
no use of fancy quotes, `'` is fine

Reply:
iʼd say just «quotes» 😅. (unicode — your days are still to come! 😆)